libs/remotion/forms/src/lib/backgrounds/ImageBackground.form.tsx

```diff
@@ -1,19 +1,36 @@
-import React from 'react';
-import { Box, Button, FormControl, FormHelperText, FormLabel, Input } from '@chakra-ui/react';
+import React, { useState } from 'react';
+import { Box, Button, ButtonGroup, FormControl, FormHelperText, FormLabel, Input, InputGroup, InputRightElement, Text, VStack } from '@chakra-ui/react';
 import { BackgroundComponents } from '@pulsebanner/remotion/components';
 import { LayerForm } from '../LayerForm';
 import { StarIcon } from '@chakra-ui/icons';
 
 export const ImageBackground: LayerForm<typeof BackgroundComponents.ImageBackground> = ({ props, setProps, showPricing, availableFeature }) => {
     console.log('show pricing: ', availableFeature);
+    const [currentString, setCurrentString] = useState(props.src ?? '');
```

Comment:
Don't keep this state in the form, instead use the `setProps` method like I was doing before. If you don't use `setProps`, the value will never be saved to the database when the user clicks `Save settings`.

Reply:
We do the setProps in the Set Image URL button for this